### DISCUSSION TITLE
Potree + Cesium + iTowns ?

### DISCUSSION THREAD

**Discussion by loicroybon** [iTowns/itowns#1587] [Q&A] [answered]
Bonjour,

_Note: je suis conscient de citer Cesium, concurrent de iTowns, dans ma question. Mes développements ayant commencé sur Cesium, je chercher ici à voir s'il est possible de minimiser les modifications dans mon projet._ 

J'utiliser actuellement un viewer Potree pour afficher mes nuages de points.
J'ai récemment intégré Cesium dans le potree_viewer en suivant cet exemple : http://potree.org/potree/examples/cesium_retz.html
Je cherche désormais à intégrer les flux MNT de l'IGN (RGE ALTI 1M) mais je découvre que Cesium n'est pas compatible avec les flux WMS format X-BIL 32 de l'IGN... 

1. Comment faire pour que Cesium puisse lire ces MNT ?
2. Si ce n'est pas possible : est-il possible de cumuler Cesium + iTowns + Potree ?
3. Si oui comment ? Si non : avez-vous d'autres idées ? Des exemples d'intégration de **iTowns dans Potree** **(et pas l'inverse !)** ?

**Comment by gchoqueux** [comment, marked as answer]
> Comment faire pour que Cesium puisse lire ces MNT ?

I don't know how use xbil DEM in Cesium, I thinks it's not possible.

> Si ce n'est pas possible : est-il possible de cumuler Cesium + iTowns + Potree ?

it's not possible to handle DEM with itowns in Cesium.

> Des exemples d'intégration de iTowns dans Potree

it's possible to use Potree with itowns. 
I took part in the development of a website (https://alex.ign.fr/visu3d), it uses Potree and iTowns. 
Soon I will add an example to explain how to use Potree in itowns.
